### ISSUE TITLE
[A11y][Link][iOS] Wrong vocalization of Link with Voice Over

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#447]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iphone 14+

### OS Version

iOS26

### App version

0.6.0 alpha (441)

### Describe the issue

on iOS: 
Link alone is read "label, link" : correct
Link with label + icon is read : "Label, **image**" : incorrect, the role "link" is missing and the icon should not be read
Link with chevron is read "Label, link" **but the chevron is also focusable** : incorrect

### Expected Behavior

Link shall always be read as a link, and associated icon or chevron should be hidden

**Comment by nouha06** [comment]
The issue has been fixed and the ticket has been closed, as we cannot release a new version. That’s why the TestFlight blocking label was applied.

**Comment by pya35** [comment]
fixed on 0.6.0 (489)
